src/extractor/index.ts

```diff
@@ -69,11 +82,43 @@ function findPossibleKeys(content: string) {
   return res;
 }
 
-export default async function extractKeys(
+export async function extractKeysFromCode(extractor: string, code: string) {
```

Comment:
This seems unused.

Reply:
It is, just haven't worked on updating the command yet so it's still here for the time being